README.md

```diff
@@ -8,7 +8,7 @@ For example, a Public Private Partnership project to build a bridge may set out
 
 The tariff extension allows a structured list of these charges to be set out.
 
-It also includes an additional codelist entries for the documentType codelist for:
+You can also find entries in the standard [documentType codelist](https://standard.open-contracting.org/latest/en/schema/codelists/#document-type) for:
```

Comment:
I suggest removing this paragraph and the list of codes.

Reply:
Why? It's convenient for the user of the extension to know that they can also use documentType codes related to tariffs. 

It's indeed a good idea to remove the list of codes so that we don't need to maintain it.

Reply:
In general, we avoid putting documentation in extensions that just explains the standard, because then it is very easy to forget that the documentation exists, and we end up with issues similar to https://github.com/open-contracting/standard/issues/707 and https://github.com/open-contracting/standard/issues/701 where we need to reconcile the two.